### PR TITLE
More mulled-build-files fixes.

### PR DESCRIPTION
**Pull request by jmchilton**
- Add explicit option ``--check-published`` to mulled-build-*.
- Fix auto-installation of Involucro on first attempt.
- Handle explicit tags in mulled-build-files and add an implicit ``1`` tag if none found.
- Fix tab parsing in mulled-build-files.
- Various other fixes.

With these changes - the [Travis setup](https://travis-ci.org/jmchilton/multireqcontainers) in [jmchilton/multireqcontainers](https://github.com/jmchilton/multireqcontainers) seems to work properly.

Ping @bgruening.

### CODE REVIEW

File: galaxy/tools/deps/mulled/mulled_build.py [hunk 1]
```diff
@@ -138,15 +138,21 @@ def mull_targets(
...
-
+    if not rebuild or "push" in command:
+        repo_name = repo_template_kwds["image"].split(":", 1)[0]
+        repo_data = quay_repository(repo_template_kwds["namespace"], repo_name)
         if not rebuild:
-            if "error_type" not in repo_data and "tags" in repo_data and repo_data["tags"]:
+            tags = repo_data.get("tags", [])
+
+            target_tag = None
+            if ":" in repo_template_kwds["image"]:
```
Comment (bgruening) [inline galaxy/tools/deps/mulled/mulled_build.py]: should we fail if there are more then one `:`

**Comment by bgruening** [comment]
GitHub renders tsv files nicely, if we include empty columns.

**Comment by jmchilton** [comment]
Yeah - good plan - I'll rework this to start at zero and put stronger condition on the image name.

Thanks for the review!

**Comment by bgruening** [comment]
Thanks John!

**Pull request merged**